cmake/XNNPACKConfig.cmake.in

```diff
@@ -0,0 +1,5 @@
+@PACKAGE_INIT@
+INCLUDE(CMakeFindDependencyMacro)
+FIND_DEPENDENCY(pthreadpool REQUIRED)
```

Comment:
What about other dependencies (e.g. `cpuinfo`)?

Reply:
Since `cpuinfo` is linked privately

https://github.com/google/XNNPACK/blob/0961e3449544d147a8486f43a3fdbb4e5d66e162/CMakeLists.txt#L832

we're signaling to CMake that `cpuinfo` is a build-time only dependency -- an implementation detail of the project that consumers of the project don't need to worry about.

Conversely, because `pthreadpool` is linked publicly

https://github.com/google/XNNPACK/blob/0961e3449544d147a8486f43a3fdbb4e5d66e162/CMakeLists.txt#L864

we're signaling to CMake that `pthreadpool` is a run-time dependency and that anyone who wants to use `XNNPACK` should make sure `pthreadpool` is available.